sklearn/svm/_base.py

```diff
@@ -148,6 +148,13 @@ def fit(self, X, y, sample_weight=None):
         If X is a dense array, then the other methods will not support sparse
         matrices as input.
         """
+        # changing default value of nu in OneClassSVM, remove in 0.26
+        if self.nu == 'warn':
+            warnings.warn('The default value of nu will change from 0.5 to 0.1'
+                          ' in version 0.26.', FutureWarning)
```

Comment:
```suggestion
                          ' in version 1.1 (renaming of 0.26).', FutureWarning)
```

Reply:
actually, now that we have release 0.24.0 without this PR, this should probably be 1.2 instead of 1.1.